starter-code/index.js

```diff
@@ -1,11 +1,46 @@
 class SortedList {
-    constructor() {}
-    add(item) {}
-    get(pos) {}
-    max() {}
-    min() {}
-    average() {}
-    sum() {}
+
+    constructor() {
+      this.items=[];
+      this.length=0;
+    }
+
+    add(item) {
```

Comment:
Il manque la méthode sort() pour classer tes item : 
  add(item) {
    this.items.push(item);
    this.items.sort(function(a, b) {
      if (a < b) return -1;
      if (a > b) return 1;
      if (a === b) return 0;
    });
    this.length++;
  }